### PR TITLE
frontend: calendar view - switch from using dummy data to backend data #54

### PR DESCRIPTION
**Pull request by 12cassie34**

### CODE REVIEW

File: client/src/maps/DateMaps.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "monday": { value: "Mon" },
+    "tuesday": { value: "Tue" },
+    "wednesday": { value: "Wed" },
+    "thursday": { value: "Thu" },
+    "friday": { value: "Fri" },
+    "saturday": { value: "Sat" },
+    "sunday": { value: "Sun" },
+});
+
+const extractDate = (date: number) => {
```
Comment (abdulqshabbir) [inline client/src/maps/DateMaps.ts]: Code looks good so far. Here is minor improvement idea. The English language is very strange and so there are some exceptions to these rules. I've copied a message from Stack Overflow that explains it well.

>st is used with numbers ending in 1 (e.g. 1st, pronounced first)
nd is used with numbers ending in 2 (e.g. 92nd, pronounced ninety-second)
rd is used with numbers ending in 3 (e.g. 33rd, pronounced thirty-third)
As an exception to the above rules, all the "teen" numbers ending with 11, 12 or 13 use -th (e.g. 11th, pronounced eleventh, 112th, pronounced one hundred [and] twelfth)
th is used for all other numbers (e.g. 9th, pronounced ninth).

source: https://stackoverflow.com/questions/13627308/add-st-nd-rd-and-th-ordinal-suffix-to-a-number

Comment (12cassie34) [reply client/src/maps/DateMaps.ts]: Thanks Abdul, I didn't know that. 😂 Will change it tonight.

**Review by cherylli — COMMENTED** [review]
Looks good. Can probably delete the dummy data and utils/getColorBySubject since we decided to just hardcode the colours now.

Also, Empty days should probably still have a width. (public holiday/school holiday) 
[](https://i.imgur.com/Chw3uJp.png)

For weeks which span 2 different months do we want to add the second month as well or are we not worrying about it like this week would be August/September instead of just August for example.

**Comment by amirfakhrullah** [comment]
Before merging, can you help me out to pull the latest code from main branch and merge to this branch. And then run yarn format

**Review by cherylli — APPROVED** [review]
looks good

**Review by abdulqshabbir — APPROVED** [review]
looks good!

**Pull request merged**